stout/event-loop.h

```diff
@@ -202,7 +202,7 @@ class EventLoop : public Scheduler {
   //
   // NOTE: takes ownership of event loop pointer.
```

Comment:
This comment is out of date now that `DefaultReset` doesn't take a pointer anymore.

Reply:
Fixed!